.github/workflows/coveralls.yml

```diff
@@ -0,0 +1,39 @@
+name: Tests and coveralls
+
+on: ["push", "pull_request"]
+
+jobs:
+  build:
+
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+
+    # Check out Housekeeper code
+    - uses: actions/checkout@v1
+
+    # Set up python
+    - name: Set up Python ${{ matrix.python-version}}
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Production Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .
+
+    - name: Development Dependencies
+      run: |
+        pip install -r requirements-dev.txt
+        pip install pytest-cov
+        pip install coveralls
```

Comment:
Install the deps on one line to make sure it considers all requirements at the same time?

ditto in the following installs

Reply:
I changed so that the check is run before the extra packages are installed. I figure this makes sense since the `requirements-dev` are necessary for development. The other are only used in CI